src/grpc/iot_config/hpr_route_ets.erl

```diff
@@ -74,18 +85,24 @@ delete_route(Route) ->
     DevAddrEntries = ets:select_delete(?ETS_DEVADDR_RANGES, MS1),
     MS2 = [{{'_', RouteID}, [], [true]}],
     EUISEntries = ets:select_delete(?ETS_EUI_PAIRS, MS2),
-    MS3 = [{{'_', {'_', RouteID, '_'}}, [], [true]}],
-    SKFsEntries = ets:select_delete(?ETS_SKFS, MS3),
+    case ?MODULE:lookup_route(RouteID) of
+        [{_, SKFETS}] ->
+            ets:delete(SKFETS);
+        _Other ->
+            lager:warning("failed to delete skf table ~p for ~s", [
+                _Other, RouteID
+            ])
+    end,
     true = ets:delete(?ETS_ROUTES, RouteID),
-    lager:info("deleted ~w DevAddr Entries, ~w EUIS Entries, ~w SKFs Entries for ~s", [
-        DevAddrEntries, EUISEntries, SKFsEntries, RouteID
+    lager:info("deleted ~w DevAddr Entries, ~w EUIS Entries for ~s", [
```

Comment:
Do we not want to use the metadata anymore in logs? I liked that it made it easier to grep for messages.
```
lager:info(
  [{devaddr, DevAddrEntries}, {euis, EUIEntries}, {route_id, RouteID}],
  "route deleted"
),
```

Reply:
no you are right, fixing